test/e2e/serial/tests/configuration.go

```diff
@@ -1067,7 +1067,15 @@ var _ = Describe("[serial][disruptive] numaresources configuration management",
 			By("checking the deployment pod has failed scheduling and its at the pending status")
 			pods, err := podlist.With(fxt.Client).ByDeployment(ctx, updatedDeployment)
 			Expect(err).ToNot(HaveOccurred())
-			Expect(len(pods)).To(Equal(1))
+			Expect(len(pods)).To(BeNumerically("<", 10), "")
```

Comment:
`, "")` is used to add details upon failure of the Expect, if you don't want to add more details please delete it

Reply:
also please add a comment explaining the reasoning for using the number 10, you can tell that pods with TAE may be created but their count should be always bounded and no more than 10 based on the results of several runs

Reply:
+1, make it a constants in this source file if possible

Reply:
alright will fix that now thanks for noticing that